src/main/resources/schemas/exchange/exchange.xsd

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            xmlns:org="urn:gov:hhs:fha:nhinc:exchange:directory"
+            xmlns:tns="urn:gov:hhs:fha:nhinc:exchange"
+            targetNamespace="urn:gov:hhs:fha:nhinc:exchange"
+            elementFormDefault="qualified">
+  <xsd:import namespace="urn:gov:hhs:fha:nhinc:exchange:directory" schemaLocation="organizationDirectory.xsd"/>
+
+  <xsd:element name="ExchangeInfo" type="tns:exchangeInfoType"/>
+  <xsd:complexType name="exchangeInfoType">
+    <xsd:sequence>
+      <xsd:element name="refreshInterval" type="xsd:long"/>
+      <xsd:element name="maxNumberOfBackups" type="xsd:integer"/>
+      <xsd:element name="defaultExchange" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="exchanges" type="tns:exchangesType"/>
+    </xsd:sequence>
+  </xsd:complexType>
+
+  <xsd:complexType name="exchangesType">
+    <xsd:sequence>
+      <xsd:element name="exchange" type="tns:exchangeType" maxOccurs="unbounded"/>
+    </xsd:sequence>
+  </xsd:complexType>
+
+  <xsd:complexType name="exchangeType">
+    <xsd:sequence>
+      <xsd:element name="tLSVersions" type="tns:tLSVersionType"/>
+      <xsd:element name="url" type="xsd:anyURI"/>
+      <xsd:element name="key" type="xsd:string" minOccurs="0"/>
+      <xsd:element name="Organization" type="org:organization" maxOccurs="unbounded"/>
+    </xsd:sequence>
+    <xsd:attribute name="type" type="xsd:string"/>
+  </xsd:complexType>
+
+   <xsd:complexType name="tLSVersionType">
```

Comment:
What's the difference between this and the tlSVersions type?  Also the S doesn't need to be capitalized for this variable, could be tlsVersions everywhere you have it.